src/instructlab/lab.py

```diff
@@ -1002,6 +1002,13 @@ def train(
             shutil.copy(file, final_results_dir)
             print("Copied ", file, "to ", final_results_dir)
 
+        if four_bit_quant:
+            print(
+                "SKIPPING CONVERSION to gguf. This is unsupported with --4-bit-quant. "
+                + "See https://github.com/instructlab/instructlab/issues/579."
+            )
+            return
```

Comment:
Why return and not make this part of the condition?

Reply:
mainly because I prefer reducing levels of nesting -- style choice